packages/node/config/localhost.json

```
@@ -1,3 +1,4 @@
 {
-  "host": "http://localhost:3003"
+  "host": "http://localhost:3003",
+  "readmeApiUrl": "https://dash.readme.local:3000/api/"
```

Haven’t looked through everything or tested this yet, but I’m real pumped for this. One small change:

```suggestion
  "readmeApiUrl": "http://dash.readme.local:3000/api/"
```